scripts/start.sh

```diff
@@ -24,36 +24,42 @@ iptables -A FORWARD -m state --state ESTABLISHED,RELATED -j ACCEPT
 
 iptables -t nat -A POSTROUTING -s 10.8.0.0/24 -o eth0 -j MASQUERADE
 
+LOCKFILE=/etc/openvpn/.gen
+
+# Regenerate certs only on the first start 
+if [ ! -f $LOCKFILE ]; then
```

Comment:
This place has a bug! 

We persist data in `/etc/openvpn` but PKI infrastructure is located in `/usr/share/easy-rsa/pki/`. 
Thus, when we rerun our container using `docker restart openvpn` we get the error:
```
Fri Feb 21 22:08:08 2020 Initialization Sequence Completed
cp: can't stat 'pki/private/client.key': No such file or directory
cp: can't stat 'pki/issued/client.crt': No such file or directory
cp: can't stat 'pki/ca.crt': No such file or directory
cat: can't open 'client/ca.crt': No such file or directory
cat: can't open 'client/client.crt': No such file or directory
cat: can't open 'client/client.key': No such file or directory
Fri Feb 21 22:08:08 2020 Client.ovpn file has been generated
```
This happens because we see that LOCKFILE has already been created and and omit creation of PKI infrustructure. But when we later start creating client configuration we bump into this error.

I think we need to persist PKI infrastructure on the volume as well

Reply:
Why do we need to re-generate pkis for the second time, when the configuration has already settled?

Reply:
Sorry, my fault. The bug I mentioned reproduces when you call `docker run` command second time without removing the volume, which I think isn't intended use.

Reply:
I'm going to fix conflicts and mere PR this week. Thanks

Reply:
@alekslitvinenk To my mind, this is the exactly desired behavior to restart the container with persisted data and let it return to the previous state. I think you were correct and this is kinda bug. I see the same log entries on my test setup. Yet, it doesn't affect server work by itself. If I got you right, you use /pki for initial keys and user config generation, right? if so, then just putting config generation step under the same `$LOCKFILE` guard will do the trick for us.

Reply:
Yes @romansavrulin, I generate **PKI** infrastructure and then build from it server and client keys. I recently also added the possibility to create additional users by calling 
```bash
docker exec dockovpn ./genclient.sh
```
Without persisting this PKI infrastructure many things can go awry now. But I'll add it as a separate PR.
Sorry for slow progress on this PR: too busy on my dayjob. I will merge it hopefully tonight